src/EdFi.AnalyticsMiddleTier.DataStandard2/Base/MSSQL/0021-View-ClassPeriodDim-Create.sql

```diff
@@ -0,0 +1,31 @@
+﻿-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+IF EXISTS (SELECT 1 FROM INFORMATION_SCHEMA.VIEWS WHERE TABLE_SCHEMA = 'analytics' AND TABLE_NAME = 'ClassPeriodDim')
+BEGIN
+	DROP VIEW analytics.ClassPeriodDim
+END
+GO
+
+CREATE VIEW analytics.ClassPeriodDim AS
+    SELECT 
+       CONCAT(Section.ClassPeriodName, '-', Section.LocalCourseCode, '-', Section.SchoolId, '-', Section.SchoolYear, '-', Section.TermDescriptorId, '-', Section.UniqueSectionCode, '-', Session.SessionName) ClassPeriodKey, 
+       Section.ClassPeriodName, 
+       Section.LocalCourseCode, 
+       Section.SchoolId, 
+       Section.SchoolYear, 
+       Section.TermDescriptorId, 
```

Comment:
**Question**. 
On the last (Friday afternoon) comment from Stephen on the Jira tickete, it says:
_In Data Standard 2.2, we must use the TermDescriptorId in the inner join
However, there is no (current) requirement to include it in the output_
---
Do we need to remove this field from the output?